### DISCUSSION TITLE
Facing issue in ingestion

### DISCUSSION THREAD

**Discussion by ankit289** [Sunbird-cQube/community#846] [General]
![image](https://github.com/Sunbird-cQube/community/assets/43002190/945e7baa-7e84-4851-ab6e-a00d1e86b863)

**Comment by RNarmatha** [comment]
@bommalla- Could you please look into this issue

**Comment by bommalla** [comment]
@ankit289 @RNarmatha 

Actually this issue is regarding  config.json syntax issue.yesterday I gave some inputs on this.

**Comment by RNarmatha** [comment]
Thanks Rajesh

@ankit289 - Could you please update the status of this ticket
